libs/application/templates/driving-license/src/forms/application.tsx

```diff
@@ -53,14 +55,15 @@ export const application: Form = buildForm({
               subTitle:
                 'Til þess að tryggja að notandi hafi heimild til þess að sækja um ökuskírteini út frá punktastöðu',
             }),
+            buildDataProviderItem({
+              id: 'residence',
+              type: undefined,
```

Comment:
I don't fully understand why we need a untyped data provider for this. Can you elaborate?

Reply:
We want a singular place where the user accepts some terms. These terms don't always fetch data but we still want the user to accept them, in this case búseta.

Reply:
This needs a further refactor where we rename this field in another slice